src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Publish.targets

```
@@ -93,7 +93,10 @@
     <!-- replace apphost with binary we generated during native compilation -->
     <Delete Files="$(PublishDir)\$(TargetName)$(NativeBinaryExt)" />
     <Copy SourceFiles="$(NativeOutputPath)$(TargetName)$(NativeBinaryExt)" DestinationFolder="$(PublishDir)" />
+  </Target>
 
+  <Target Name="_CopyAotSymbols" AfterTargets="CopyNativeBinary"
```

```suggestion
  <Target Name="_CopyAotSymbols" AfterTargets="Publish"
```

It feels like someone deleting `CopyNativeBinary` without searching for all the places that refer to this string is a thing that could happen. This would prevent it.

---

> It feels like someone deleting CopyNativeBinary without searching for all the places that refer to this string is a thing that could happen. 

Tests would prevent that.

---

I will add a test for this in the SDK repo.

---

But it's a good suggestion regardless, why not parallelize?